ui/src/app/kvm/views/LXDClusterDetails/LXDClusterSettings/LXDClusterSettings.tsx

```diff
@@ -14,8 +18,17 @@ const LXDClusterSettings = ({
   clusterId,
   setHeaderContent,
 }: Props): JSX.Element => {
+  const cluster = useSelector((state: RootState) =>
+    vmClusterSelectors.getById(state, clusterId)
+  );
+  useActivePod(cluster?.hosts[0].id || null);
+
   return (
-    <Strip className="u-no-padding--top" shallow>
+    <Strip shallow>
+      <AuthenticationCard
+        hostId={cluster?.hosts[0].id || null}
```

Comment:
I think we might want to be a little more defensive here just in case there are no hosts (which I don't think should be possible, but you never know with clusters).

Reply:
Agreed, thanks